test/e2e/spec/eid_oidc_provider/jwks_spec.js

```diff
@@ -0,0 +1,32 @@
+describe("OpenId: JWKS", function() {
+    
+    var chakram = require('chakram');
+    var expect = chakram.expect;
+    var URL = 'https://' + process.env.BOILERPLATE_DOMAIN + '/api/openid';
+
+    it("provides JWK for vailid key id", function() {
+        // TODO get valid key id
```

Comment:
@BenjaminKeller Please tell me what you think about this. Travis is failing because of the TODO. I assume that this function will get a valid key id when you add the fake authentication to the db?

Reply:
That's cool. I will add the required functionality.